tutorials/how_to/laziness.md

```diff
@@ -0,0 +1,31 @@
+# How to Train a Model with Lazy Data
+
+(Here, by "lazy data", I mean a dataset that's generated
+ instance-by-instance each time you iterate over it,
+ possibly because it's too big to fit in memory,
+ possibly because you want to sample your data each iteration,
+ possibly because you want to start training immediately,
+ possibly for some other reason I haven't thought of.)
+
+There are basically four things you should do. This tutorial will tell you
+what they are. If you want to know the details, you should check out the
+[detailed tutorial about laziness in AllenNLP](../getting_started/laziness.md).
+
+## In the `YourDatasetReader` constructor
+
+`YourDatasetReader` subclass should have a `lazy` parameter in its constructor
+and `from_params` method, and pass that value to its superclass constructor.
+
+## In `YourDatasetReader._read()`
+
+`YourDatasetReader._read()` should return a generator, not a list.
```

Comment:
Is it possible to link to particular code blocks in the other tutorial for `generator` and `list`, in case someone doesn't know what this means?  Worst case, you could just link to the same thing for `generator` that you did in the tutorial.